test/fixtures/dialects/ansi/create_table_column_constraint.yml

```diff
@@ -129,12 +130,15 @@ file:
               expression:
               - column_reference:
                   identifier: Age
-              - comparison_operator: '>='
+              - comparison_operator:
+                - raw_comparison_operator: '>'
+                - raw_comparison_operator: '='
               - literal: '18'
               - binary_operator: AND
               - column_reference:
                   identifier: City
-              - comparison_operator: '='
+              - comparison_operator:
+                  raw_comparison_operator: '='
```

Comment:
good example of consistent AST. rules can just refer to `is_type("comparison_operator")` and use the name to decide whether its greater than, less than as before 👍